catalog/task/push-sbom-to-pyxis/0.1/push-sbom-to-pyxis.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: push-sbom-to-pyxis
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that extracts sbom from a pull spec and pushes it to Pyxis.
+  params:
```

Comment:
It can be done in the later PR, but you are going to need stuff in here for authenticating to pyxis and which endpoint to use. As long as you are aware of that, if you are choosing to do it later on, you can resolve this

Reply:
Yes, my intention was to add it when it's needed - in https://issues.redhat.com/browse/HACBS-1553
And it will likely get inspired by how you end up doing it in
https://github.com/hacbs-release/release-bundles/pull/72